packages/components/select/src/useSelect.ts

```diff
@@ -577,7 +578,7 @@ export const useSelect = (props, states: States, ctx) => {
   const handleResize = () => {
     resetInputWidth()
     tooltipRef.value?.updatePopper?.()
-    if (props.multiple) resetInputHeight()
+    props.multiple && nextTick(() => resetInputHeight())
```

Comment:
`resetInputHeight` has used nextTick internally, is it necessary to add nextTick here?

Reply:
done.